### PR TITLE
Fix typo in csc help text

### PR DESCRIPTION
**Pull request by tmat**

### CODE REVIEW

**Comment by tmat** [comment]
@dotnet/roslyn-compiler

File: src/Compilers/CSharp/Portable/CSharpResources.resx [hunk 1]
```diff
@@ -4411,7 +4411,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
                                generation but may be used by analyzers for producing
                                errors or warnings.
  /embed                        Embed all source files in the PDB.
- /embed:&lt;file list&gt;            Embed specfic files the PDB
+ /embed:&lt;file list&gt;            Embed specific files the PDB
```
Comment (cston) [inline src/Compilers/CSharp/Portable/CSharpResources.resx]: `... in the PDB`?

Comment (tmat) [inline src/Compilers/CSharp/Portable/CSharpResources.resx]: Yes :)

**Comment by cston** [comment]
LGTM

**Pull request merged**